client/src/app/site/motions/modules/motion-detail/components/motion-detail/motion-detail.component.ts

```diff
@@ -226,17 +225,17 @@ export class MotionDetailComponent extends BaseModelContextComponent implements
         this.getMotionByUrl();
 
         // load config variables
-        this.meetingSettingsService
+        this.meetingSettingService
```

Comment:
Why does it work, since you deleted it in the constructor?

Reply:
The `BaseComponent` provides the `meetingSettingService`.

Reply:
Ok, thanks!